common/src/thrift/tachyon.thrift

```diff
@@ -220,6 +220,9 @@ service FileSystemMasterService {
   void setPinned(1: i64 fileId, 2: bool pinned)
     throws (1: TachyonTException e)
 
+  void setTTL(1: i64 fileId, 2: i64 ttl)
```

Comment:
Sounds like a good idea, but how can we determine whether a specific state needs to be set? One way is to have a `boolean` indicating whether to set the state for each state, another way is to use `list/set` of the base type for each state, then empty list/set indicates that do not set the state. Both are not as straightforward as the current solution.